### PR TITLE
Run ReSharper for general cleanup

### PR DESCRIPTION
**Pull request by ethax-ross**

### CODE REVIEW

File: GetIntoTeachingApi/Auth/SharedSecretHandler.cs [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 using Microsoft.AspNetCore.Http;
-using System;
 using System.Threading.Tasks;
 
 namespace GetIntoTeachingApi.Auth
 {
     public class SharedSecretHandler : AuthorizationHandler<SharedSecretRequirement>
     {
-        IHttpContextAccessor _httpContextAccessor;
+        private readonly IHttpContextAccessor _httpContextAccessor;
```
Comment (DrLazer) [inline GetIntoTeachingApi/Auth/SharedSecretHandler.cs:9]: would defo not have this as readonly

File: GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs [hunk 2]
```diff
@@ -46,7 +45,7 @@ ICrmService crm
                 return BadRequest(this.ModelState);
             }
 
-            return Ok(new Object());
+            return Ok(new object());
```
Comment (DrLazer) [inline GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs:48]: conventionally caps, resharper is wrong

Comment (DrLazer) [reply GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs:48]: ill not comment on each instance of Object

File: GetIntoTeachingApi/Models/ExistingCandidateRequest.cs [hunk 3]
```diff
@@ -8,13 +7,9 @@ public class ExistingCandidateRequest
     {
         private static readonly int MinimumAdditionalAttributeMatches = 2;
 
-        [SwaggerSchema("First name")]
```
Comment (DrLazer) [inline GetIntoTeachingApi/Models/ExistingCandidateRequest.cs:11]: are annotations no longer needed for swagger?

File: GetIntoTeachingApiTests/Models/CandidateQualificationTests.cs [hunk 4]
```diff
@@ -11,11 +11,16 @@ public class CandidateQualificationTests
...
+            var entity = new Entity
+            {
+                Id = Guid.NewGuid(),
+                Attributes =
+                {
+                    ["dfe_category"] = new OptionSetValue {Value = 1},
+                    ["dfe_type"] = new OptionSetValue {Value = 2},
+                    ["dfe_degreestatus"] = new OptionSetValue {Value = 3}
+                }
+            };
```
Comment (DrLazer) [inline GetIntoTeachingApiTests/Models/CandidateQualificationTests.cs:23]: I would avoid inline construction like this ... the debugger cannot drop into this, it sees it as one line

Comment (DrLazer) [reply GetIntoTeachingApiTests/Models/CandidateQualificationTests.cs:23]: again, ill not comment on each instance

File: GetIntoTeachingApi/Models/ExistingCandidateRequest.cs (hunk 3 above)
Comment (ethax-ross) [reply GetIntoTeachingApi/Models/ExistingCandidateRequest.cs:11]: They are, but I've decided to only put them in when they actually add some value... this seemed a little pointless on reflection 🙄

File: GetIntoTeachingApiTests/Models/PrivacyPolicyTests.cs [hunk 5]
```diff
@@ -11,9 +11,7 @@ public class PrivacyPolicyTests
         [Fact]
         public void Constructor_WithEntity_MapsCorrectly()
         {
-            var entity = new Entity();
-            entity.Id = Guid.NewGuid();
-            entity.Attributes["dfe_details"] = "text";
+            var entity = new Entity {Id = Guid.NewGuid(), Attributes = {["dfe_details"] = "text"}};
```
Comment (DrLazer) [inline GetIntoTeachingApiTests/Models/PrivacyPolicyTests.cs:15]: the way you had it was much clearer

File: GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs (hunk 2 above)
Comment (ethax-ross) [reply GetIntoTeachingApi/Controllers/TeacherTrainingAdviser/CandidatesController.cs:48]: These are temporary anyway - they'll be updated to specific objects in future PRs

File: GetIntoTeachingApi/Auth/SharedSecretHandler.cs (hunk 1 above)
Comment (ethax-ross) [reply GetIntoTeachingApi/Auth/SharedSecretHandler.cs:9]: Why?

**Pull request merged**